### PR TITLE
Updated hcl file from last apply

### PR DESCRIPTION
**Pull request by gammazero**

### CODE REVIEW

File: deploy/infrastructure/dev/us-east-2/.terraform.lock.hcl [hunk 1]
```diff
@@ -3,8 +3,9 @@
 
 provider "registry.terraform.io/hashicorp/aws" {
   version     = "4.7.0"
-  constraints = ">= 3.63.0, >= 3.72.0"
+  constraints = ">= 2.23.0, >= 2.49.0, >= 2.53.0, >= 3.28.0, >= 3.63.0, >= 3.72.0"
```
Comment (masih) [inline deploy/infrastructure/dev/us-east-2/.terraform.lock.hcl:6]: which version of terraform are you using locally?

Comment (gammazero) [reply deploy/infrastructure/dev/us-east-2/.terraform.lock.hcl:6]:
```
terraform version
Terraform v1.3.6
on darwin_arm64
+ provider registry.terraform.io/hashicorp/aws v4.7.0
+ provider registry.terraform.io/hashicorp/cloudinit v2.2.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.10.0
+ provider registry.terraform.io/hashicorp/tls v3.1.0
```

Comment (masih) [reply deploy/infrastructure/dev/us-east-2/.terraform.lock.hcl:6]: I can't reproduce this lock file change locally but the changes seem harmless.

**Review by masih — APPROVED** [review]

File: deploy/infrastructure/dev/us-east-2/.terraform.lock.hcl (hunk 1 above)
Comment (masih) [reply deploy/infrastructure/dev/us-east-2/.terraform.lock.hcl:6]: Did you have a pre-existing `.terraform` folder which was created by an old version of `terraform`? 

Can I ask you to try deleting `.terraform` on your machine, pull the lock from `main` and see if running `terraform init` changes the lock again please?

**Pull request merged**